src/main/java/com/fvv/bookstore/controller/DvdControllerImpl.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * 
+ */
+package com.fvv.bookstore.controller;
+
+import com.fvv.bookstore.bean.Dvd;
+import com.fvv.bookstore.bean.MovieDvd;
+
+/**
+ * 
+ * 
+ * @author Fatima Vasquez
+ * Created on 24 de mar de 2018	
+ * @version 1.0 	
+ *
+ */
+public class DvdControllerImpl {
+	
+	public boolean addDvd(Dvd dvd) {
```

Comment:
Since this is a sample code, not the real one and this is going to be merged into master. please add a comment like

```
/*
* TODO: This is a temporary implementation 
*/
```

Reply:
Ok!